frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-auth-methods.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import { Trans } from 'react-i18next';
+
+import {
+  InputGroup,
+  FormGroup,
+  TextInput,
+  Button,
+  ValidatedOptions,
+  Tooltip,
+} from '@patternfly/react-core';
+import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import { VaultConfig } from '../../types';
+
+export const VaultTokenConfigure: React.FC<VaultAuthMethodProps> = ({
+  t,
+  className,
+  vaultState,
+  setAuthValue,
+  isValid,
+}) => {
+  const [revealToken, setRevealToken] = React.useState(false);
+
+  return (
+    <FormGroup
+      fieldId="vault-token"
+      label={t('ceph-storage-plugin~Token')}
+      className={`${className}__form-body`}
```

Comment:
```suggestion
      className={className}
```
Consumers of this component are less likely to break BEM convention if we don't append element tags internally.

Reply:
ack